recipes/zstd/all/conanfile.py

```
@@ -35,7 +30,7 @@ def config_options(self):
 
     def configure(self):
         del self.settings.compiler.libcxx
-        del self.settings.compiler.libstd
```

interesting, we might need to improve hook to detect such misspellings

---

not bad idea!

---

Maybe it should be Conan itself?

---

@jgsogo Yeah, why not. If a setting is being delete and is not listed in settings.yml, then we can raise an exception.